server/routes/visit/visitUtils.ts

```diff
@@ -1,22 +1,14 @@
+import { VisitBookingDetailsDto } from '../../data/orchestrationApiTypes'
+
 // eslint-disable-next-line import/prefer-default-export
-export const getPrisonerLocation = ({
-  prisonId,
-  prisonName,
-  cellLocation,
-  locationDescription,
-}: {
-  prisonId: string
-  prisonName: string
-  cellLocation: string
-  locationDescription: string
-}) => {
-  if (prisonId === 'OUT') {
-    return locationDescription
+export const getPrisonerLocation = (prisoner: VisitBookingDetailsDto['prisoner']) => {
```

Comment:
Just refactored this to take a whole 'prisoner' rather than specifying so many parameters.